hexrd/fitting/calibration/calibrator.py

```diff
@@ -73,10 +69,8 @@ def calibration_picks(self):
         string of the hkl. And "picks" are either a list of points (powder)
         or a single point (laue). The picks are in cartesian coordinates.
         """
-        pass
 
     @calibration_picks.setter
     @abstractmethod
     def calibration_picks(self, val):
         """Setter for calibration_picks. See getter docs for details."""
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.